tools/SetupFlow/DevHome.SetupFlow/Views/QuickstartPlaygroundView.xaml

```diff
@@ -129,8 +220,8 @@
                                      PlaceholderText="{x:Bind ViewModel.PromptTextBoxPlaceholder, Mode=OneWay}"
                                      x:Name="CustomPrompt" 
                                      TextWrapping="Wrap"
-                                     AcceptsReturn="True" 
-                                     MinWidth="800"
+                                     KeyDown="CustomPrompt_KeyDown"

```

Comment:
Considering `CustomPrompt_KeyDown` calls a method on the ViewModel, consider using a `[RelayCommand] to bypass code-behind.